### ISSUE TITLE
Option '--check-features' is broken in 5.38 (r2031)

### ISSUE THREAD

**Issue by BigFatBass** [rigaya/NVEnc#367]
Hi rigaya, I've surprised the '--check-features' option doesn't display any info for the 'NVEnc features' and 'NVDec features' in 5.38.
Moreover, the command 'NVEncC64.exe --check-features' returns an unexpected non-zero status code.
It prints out only general info, e.g.:

```
.\NVEncC64.exe --check-features
NVEncC (x64) 5.37 (r2015) by rigaya, Aug 10 2021 11:05:02 (VC 1929/Win)
  [NVENC API v11.1, CUDA 10.1]
 reader: raw, y4m, avi, avs, vpy, avsw, avhw [H.264/AVC, H.265/HEVC, MPEG2, VP8, VP9, VC-1, MPEG1, MPEG4, AV1]

Environment Info
OS : Windows 10 x64 (19043) [UTF-8]
CPU: Intel Core i9-9900K @ 3.60GHz [TB: 4.49GHz] (8C/16T)
RAM: Used 12960 MB, Total 31669 MB

#0: NVIDIA GeForce GTX 970 (1664 cores, 1253 MHz)[PCIe3x16][472.12]

```
**Note**: no issue in 5.37.

Test environment:
OS: Windows 10 x64
GPU: GTX 970
Driver version: 472.12

**Comment by rigaya** [comment]
Thank you for reporting the issue, shall be fixed with [NVEnc 5.39](https://github.com/rigaya/NVEnc/releases/tag/5.39).

**Comment by BigFatBass** [comment]
Thanks a lot. Now it is working as expected in 5.39.
